perllib/FixMyStreet/Script/UpdateAllReports.pm

```diff
@@ -40,23 +42,22 @@ sub generate {
     while ( my @problem = $problems->next ) {
         my %problem = zip @cols, @problem;
         my @bodies;
+        my @areas;
         my $cobrand = $problem{cobrand};
+        my $duration_str = ( $problem{duration} > 2 * $fourweeks ) ? 'old' : 'new';
+        my $type = ( $problem{duration} > 2 * $fourweeks )
+            ? 'unknown'
+            : ($problem{age} > $fourweeks ? 'older' : 'new');
+        my $problem_fixed =
+               FixMyStreet::DB::Result::Problem->fixed_states()->{$problem{state}}
+            || FixMyStreet::DB::Result::Problem->closed_states()->{$problem{state}};
+
+        # Add to bodies it was sent to
+        @bodies = split( /,/, $problem{bodies_str} );
```

Comment:
This will issue a warning on any problem not sent to any body, needs a `next unless $problem{bodies_str}` before it or similar? Or the DB query could exclude those with null entries.